Shell/LineEditor.cpp

```diff
@@ -106,15 +111,16 @@ void LineEditor::cut_mismatching_chars(String& completion, const String& other,
     completion = completion.substring(0, i);
 }
 
-void LineEditor::tab_complete_first_token(const String& token)
+Vector<String> LineEditor::tab_complete_first_token(const String& token)
 {
```

Comment:
This looks like it could return `Vector<StringView`?

Reply:
Oh it's assigned to a `Vector<String>` at call site. Maybe it could use a comment about that :)

Reply:
Will do :P